### PR TITLE
[new release] ocaml-lsp-server, lsp and jsonrpc (1.12.3)

### PR DESCRIPTION
**Pull request by rgrinberg**
LSP Server for OCaml

- Project page: <a href="https://github.com/ocaml/ocaml-lsp">https://github.com/ocaml/ocaml-lsp</a>

## Fixes

- Fix a bad interaction between inferred interfaces and promotion code actions
  in watch mode (ocaml/ocaml-lsp#753)

- Fix URI parsing (ocaml/ocaml-lsp#739 fixes ocaml/ocaml-lsp#471 and ocaml/ocaml-lsp#459)

### CODE REVIEW

File: packages/lsp/lsp.1.12.3/opam [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  "Max Lantas <mnxndev@outlook.com>"
+]
+license: "ISC"
+homepage: "https://github.com/ocaml/ocaml-lsp"
+bug-reports: "https://github.com/ocaml/ocaml-lsp/issues"
+depends: [
+  "dune" {>= "3.0"}
+  "jsonrpc" {= version}
+  "dyn"
+  "yojson"
```
Comment (kit-ty-kate) [inline packages/lsp/lsp.1.12.3/opam:28]:
```suggestion
  "yojson"
  "uri"
```
```
#=== ERROR while compiling lsp.1.12.3 =========================================#
# context              2.1.2 | linux/x86_64 | ocaml-base-compiler.4.12.1 | pinned(https://github.com/ocaml/ocaml-lsp/releases/download/1.12.3/lsp-1.12.3.tbz)
# path                 ~/.opam/4.12/.opam-switch/build/lsp.1.12.3
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p lsp -j 31 @install
# exit-code            1
# env-file             ~/.opam/log/lsp-3565-6a88f8.env
# output-file          ~/.opam/log/lsp-3565-6a88f8.out
### output ###
# File "lsp/src/dune", line 6, characters 56-59:
# 6 |  (libraries jsonrpc ppx_yojson_conv_lib dyn uutf yojson uri)
#                                                             ^^^
# Error: Library "uri" not found.
# -> required by library "lsp" in _build/default/lsp/src
# -> required by _build/default/META.lsp
# -> required by _build/install/default/lib/lsp/META
# -> required by _build/default/lsp.install
# -> required by alias install
```

**Comment by rgrinberg** [comment]
Thanks for noticing. Do you mind fixing it here? I'll do it upstream

**Comment by rgrinberg** [comment]
@mseri @kit-ty-kate gentle ping

**Comment by kit-ty-kate** [comment]
Oops, sorry i forgot about this one. Thanks! Could you return the fixes to the opam files upstream?

**Pull request merged**